targets/pna_nic/tests/testdata/pna-demo-L2-one-table.json

```diff
@@ -0,0 +1,179 @@
+        "metadata" : true,
+        "pi_omit" : true
+      },
+      {
+        "name" : "pna_main_input_metadata",
+        "id" : 2,
+        "header_type" : "pna_main_input_metadata_t",
+        "metadata" : true,
+        "pi_omit" : true
+      },
+      {
+        "name" : "pna_main_output_metadata",
+        "id" : 3,
+        "header_type" : "pna_main_output_metadata_t",
+        "metadata" : true,
+        "pi_omit" : true
+      },
+      {
+        "name" : "<null>",
+        "id" : 4,
+        "header_type" : "scalars_t",
+        "metadata" : false,
+        "pi_omit" : true
+      }
+    ],
+    "header_stacks" : [],
+    "header_union_types" : [],
+    "header_unions" : [],
+    "header_union_stacks" : [],
+    "field_lists" : [],
+    "errors" : [],
+    "enums" : [
+      {
+        "name" : "PNA_MeterColor_t",
+        "entries" : [
+          ["GREEN", 1],
+          ["RED", 0],
+          ["YELLOW", 2]
+        ]
+      }
+    ],
+    "parsers" : [
+      {
+        "name" : "main_parser",
+        "id" : 0,
+        "init_state" : "start",
+        "parse_states" : [
+          {
+            "name" : "start",
+            "id" : 0,
+            "parser_ops" : [
+              {
+                "parameters" : [
+                  {
+                    "type" : "regular",
+                    "value" : "eth"
+                  }
+                ],
+                "op" : "extract"
+              }
+            ],
+            "transitions" : [
+              {
+                "type" : "default",
+                "value" : null,
+                "mask" : null,
+                "next_state" : null
+              }
+            ],
+            "transition_key" : []
+          }
+        ]
+      }
+    ],
+    "parse_vsets" : [],
+    "deparsers" : [
+      {
+        "name" : "main_deparser",
+        "id" : 0,
+        "source_info" : {
+          "filename" : "pna-demo-L2-one-table.p4",
+          "line" : 98,
+          "column" : 8,
+          "source_fragment" : "MainDeparserImpl"
+        },
+        "order" : ["eth"],
+        "primitives" : []
+      }
+    ],
+    "meter_arrays" : [],
+    "counter_arrays" : [],
+    "register_arrays" : [],
+    "calculations" : [],
+    "learn_lists" : [],
+    "actions" : [],
+    "pipelines" : [
+      {
+        "name" : "main_control",
+        "id" : 0,
+        "source_info" : {
+          "filename" : "pna-demo-L2-one-table.p4",
+          "line" : 58,
+          "column" : 8,
+          "source_fragment" : "MainControlImpl"
+        },
+        "init_table" : null,
+        "tables" : [],
+        "action_profiles" : [],
+        "conditionals" : []
+      }
+    ],
+    "checksums" : [],
+    "force_arith" : [],
+    "extern_instances" : [],
+    "field_aliases" : [],
+    "program" : "pna-demo-L2-one-table.p4",
+    "__meta__" : {
+      "version" : [2, 23],
+      "compiler" : "https://github.com/p4lang/p4c"
+    }
+  }
+  
```

Comment:
Github is complaining about a missing newline character, but maybe we can ignore it if this file was generated _as is_ by p4c. (Was it generated or did you write it by hand?).

Reply:
Generated by p4c-bm2-pna compiler. I will ass a new line at the end.

Reply:
not addressed?